backend/services/comment.ts

```diff
@@ -0,0 +1,14 @@
+import * as commentDAO from "../storage/comment";
+import { Comment } from '../models/comment';
+import { generateUUID } from "../util/uuid";
+
+
+export function createComment(comment: Comment): Promise<Comment> {
```

Comment:
maybe we should handle an error like -> created comment on a not existing recipe?